script/stats.py

```diff
@@ -0,0 +1,202 @@
+import json
+import pandas as pd
+from pandas import NA
+import numpy  as np
+import datetime
+from urllib.request import urlopen
+
+
+# Show all data 
+pd.set_option('display.max_columns',None)
+pd.set_option('display.max_rows',None)
+
+
+
+def dataFilter(raw_df):
+    
+    committee_list = raw_df['committee']
+    drop_list = []
+    for committee_name in committee_list:
+        sub_df = raw_df[raw_df['committee'] == committee_name] 
+        if sub_df['size'].max() < 15:
```

Comment:
Why do you check the committee size here?
Could you add some comments on what kind of work that the dataFilter does?

Reply:
This is a meaningless function that I forgot to delete.